server/base/src/main/java/org/apache/accumulo/server/AbstractServer.java

```diff
@@ -243,7 +243,9 @@ public void startServiceLockVerificationThread() {
     final long interval =
         getConfiguration().getTimeInMillis(Property.GENERAL_SERVER_LOCK_VERIFICATION_INTERVAL);
     if (interval > 0) {
-      verificationThread = Threads.createThread("service-lock-verification-thread",
+      // TODO KEVIN RATHBUN verifying the service lock is a critical function of any process
+      // calling this and the thread would not be recreated on failures
+      verificationThread = Threads.createCriticalThread("service-lock-verification-thread",
```

Comment:
verifying the service lock is a critical function of any process calling this and the thread would not be recreated on failures